js/views/baseVw.js

```diff
@@ -37,7 +37,11 @@ export default class baseVw extends View {
    */
   getFormData(selector) {
     const $formFields = selector instanceof $ ?
-      selector : this.$(selector || 'select[name], input[name], textarea[name]');
+      selector : this.$(selector ||
+        `select[name], input[name], 
+        textarea[name]:not([data-do-not-collect]), 
```

Comment:
Instead of having to add in the `data-do-not-collect` attribute, couldn't the selector just be `textarea[name]:not(.trumbowyg-textarea)`?